storybook-addons/manager.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { addons, types } from "@storybook/addons";
+import { useArgs } from "@storybook/api";
+
+const buttonStyles = {
+  background: 'none',
+  border: 'none',
+  cursor: 'pointer',
+  color: '#767676',
+  fontSize: '13px',
+  fontWeight: 'bold'
+}
+
+export const GetButton = () => {
+  const [args] = useArgs();
+  function _handleClick() {
+    const iframe = document.getElementById("storybook-preview-iframe");
+    const component = iframe.contentWindow.document.getElementById(args.id);
+    Object.keys(args).forEach(key => {
+      console.log(`"${key}" value: ${component[key]}`);
```

Comment:
If applying this script to each component and click "Get Props" button, the console log will be complex and cannot distinguish the component info.
How about making it this kind of object?
DatePicker props' value: { id: "test", value: "12-12", ...}

Reply:
I updated